client/client.go

```diff
@@ -200,7 +200,11 @@ func (client *Client) ServerVersion() (string, error) {
 		version = "unknown"
 	}
 
-	return fmt.Sprintf("%s (series %s)", version, sysInfo.Series), nil
+	if sysInfo.OnClassic {
+		return fmt.Sprintf("%s (series %s, os: %s, classic)", version, sysInfo.Series, sysInfo.OS), nil
```

Comment:
Format is getting a bit wild. How about:
```
snap   2.0.5
snapd  2.0.5 series 16
ubuntu 16.04
```

No need to say "classic". It cannot possibly be out of classic if the OS is ubuntu 16.04. The last line is ID + VERSION_ID from os-release.